client/my-sites/checkout/composite-checkout/use-get-thank-you-url.js

```diff
@@ -149,7 +150,9 @@ export function getThankYouPageUrl( {
 
 	// Display mode is used to show purchase specific messaging, for e.g. the Schedule Session button
 	// when purchasing a concierge session.
-	const displayModeParam = getDisplayModeParamFromCart( cart );
+	const displayModeParam = isWhiteGloveOffer
+		? { d: 'white-glove' }
```

Comment:
This function is fully covered by tests, so it's probably a good idea to add a test for this.